packages/web/src/app/doc-pages/brand/illustrations-doc/illustrations-exhibit/illustrations-exhibit-details/illustrations-exhibit-details.component.ts

```diff
@@ -43,25 +45,24 @@ export class IllustrationsExhibitDetailsComponent {
     this.illustrationExhibitService.setSelectedIllustration(null);
   }
 
-  imageFileName = (format: 'svg' | 'png') => {
-    if (this.colorValue() === 'grey' || !this.colorValue()) {
-      return `${this.selectedIllustration()}.${format}`;
-    }
-    return `${this.selectedIllustration()}_${this.colorValue()}.${format}`;
-  };
-
-  downloadSvg = () => {
+  downloadImage = async (format: 'svg' | 'png') => {
     const a = document.createElement('a');
-    a.href = createSvgBlobFromElement(this.getIllustrationElement(), this.colorValue());
-    a.download = this.imageFileName('svg');
+    a.href = await (format === 'svg'
+      ? createSvgBlobFromElement(this.getIllustrationElement(), this.colorValue(), {
+          isDarkTheme: this.theme() === 'dark',
+        })
+      : createPngBlob(this.getIllustrationElement(), this.colorValue(), {
+          isDarkTheme: this.theme() === 'dark',
+        }));
+    a.download = this.imageFileName(format);
     a.click();
   };
 
-  downloadPng = async () => {
-    const a = document.createElement('a');
-    a.href = await createPngBlob(this.getIllustrationElement(), this.colorValue());
-    a.download = this.imageFileName('png');
-    a.click();
+  private imageFileName = (format: 'svg' | 'png') => {
+    if (this.colorValue() === 'grey' || !this.colorValue()) {
+      return `${this.selectedIllustration()}.${format}`;
+    }
+    return `${this.selectedIllustration()}_${this.colorValue()}.${format}`;
```

Comment:
Should we also include the current theme? This will help distinguishing the images when downloading in both themes:
<img width="648" alt="Skjermbilde 2024-04-05 kl  09 15 39" src="https://github.com/3lvia/designsystem/assets/54577949/8294ed8e-c779-4970-9297-67914b8aa2e8">